client/src/components/JitsiGame.js

```diff
@@ -163,6 +236,41 @@ class JitsiGame {
         console.log(this._api._url);
     }
 
+    /**
+ * this function will make a ul of links using saved urls from db
+ *
+ **/
+    handleGameList(selector) {
+        console.log('finished!');
+        const gamelist = this.gameList();
+
+        console.log(selector);
+
+        // document.getElementById(elementId).appendChild(gameList);
+        gamelist.then(games => {
+            document.querySelector(selector).innerHTML = ' ';
+            const gameList = document.createElement('ul');
+
+            console.log(games);
+            games.forEach(game => {
+                const li = document.createElement('li');
+
+                li.classList.add('roomLink');
+                li.appendChild(document.createTextNode(game));
+                li.addEventListener('click', () => {
+                    this.newGame('#gamelist', game);
+                    this.Game.renderGame();
+                    this.Game.handleClickEvents();
```

Comment:
Same here, if renderGame calls handleClickEvents you won't need to run both functions here but instead just run renderGame.